src/crate/crash/test_commands.py

```diff
@@ -17,13 +19,23 @@
 # with Crate these terms will supersede the license and you may use the
 # software solely pursuant to the terms of the relevant commercial agreement.
 
+import os
+import shutil
+import tempfile
 from unittest import TestCase
 from mock import patch
 from .commands import ReadFileCommand, ToggleAutocompleteCommand
 
 
 class ReadFileCommandTest(TestCase):
 
+    def setUp(self):
+        self.tmp_dir = tempfile.mkdtemp()
+
+    def tearDown(self):
+        if os.path.exists(self.tmp_dir):
+            shutil.rmtree(self.tmp_dir)
```

Comment:
fyi: There is also a `ignore_errors` parameter - has the advantage that there is no race condition